yarn-project/pxe/src/contract_function_simulator/contract_function_simulator.ts

```diff
@@ -146,8 +184,15 @@ export class ContractFunctionSimulator {
    * accounts if not specified.
```

Comment:
```suggestion
   * accounts if not specified.
   * @param skipClassVerification - Whether to skip class verification.
```

This comment was in docs of `callPrivateFunction` instead of here.

Could you also please expand on why you would want to skip class verification?

Reply:
This was forgotten from an old approach, sorry. Removing now